src/pypit.py

```diff
@@ -10,7 +10,6 @@
 import traceback
 
 # Import PYPIT routines
-debug = True
```

Comment:
Do you think it might be better to import ardebug here (that way we don't have to import it twice)? The relevant lines of code will be:

import ardebug
debug = ardebug.init()
#debug['develop']=True

where the third line should be commented out when doing a PR, and uncommented when someone is developing the code. We could then remove lines 57-59 and lines 167-168 below.